examples/applications/slack/README.md

```diff
@@ -0,0 +1,19 @@
+# Slack channel witer
+
+This sample application shows how to iwrite to a slack channel
+
+
+## Deploy the LangStream application
+```
+langstream docker run slack -app examples/applications/slack -i examples/instances/kafka-docker.yaml  -s examples/secrets/secrets.yaml --docker-args -u --docker-args 0
```

Comment:
`--docker-args -u --docker-args 0` is now no more needed on Linux (yesterday we cut release 0.3.0 with a fix for it)